packages/cli/src/options/beaconNodeOptions/api.ts

```diff
@@ -89,4 +91,12 @@ export const options: CliCommandOptions<ApiArgs> = {
     type: "number",
     description: "Defines the maximum payload, in bytes, the server is allowed to accept",
   },
+
+  "rest.swaggerUI": {
+    type: "boolean",
+    description:
+      "Enable Swagger UI for API exploration at http://{address}:{port}/documentation. This should not be used in production.",
```

Comment:
> This should not be used in production.

What does this mean exactly, why should it not be used or just not enabled?

I can see this feature to be pretty useful for people using Lodestar to research, we could maybe even enable this on our public nodes, I used curl many times to query those, the a API explorer would be convenient.

Reply:
I was just wondering why this is explicitly mentioned. The software I was previously working on was mostly closed source and we also disabled the API explorers in production because it "leaked" information about all the existing APIs. This is not the case for Lodestar as it all open source and there is no implementation / information leak.

Just curious if there are other reasons why this should not be used in production